lib/data/plugins.json

```diff
@@ -1047,6 +1047,13 @@
     "repository": "https://github.com/segmentio/metalsmith-prompt",
     "description": "Prompt the user for series of answers in the command line and add them to the global metadata."
   },
+  {
+    "name": "Projects",
```

Comment:
Alphabetically, I believe this should go one entry up, right? Pro**j**ects _then_ Pro**m**pt.